src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -42,6 +42,20 @@
 
 namespace Gudhi {
 
+/**
+ * \class Extended_simplex_type Simplex_tree.h gudhi/Simplex_tree.h
+ * \brief Extended simplex type data structure for representing the type of simplices in an extended filtration.
+ *
+ * \details The extended simplex type can be either UP (which means
+ * that the simplex was present originally, and is thus part of the ascending extended filtration), DOWN (which means
+ * that the simplex is the cone of an original simplex, and is thus part of the descending extended filtration) or
+ * EXTRA (which means the simplex is the cone point).
+ *
+ * Details may be found in section 2.2 in https://link.springer.com/article/10.1007/s10208-017-9370-z.
+ *
+ */
+enum class Extended_simplex_type {UP, DOWN, EXTRA};
```

Comment:
For some reason this does not appear in the documentation :confused:

Reply:
Are you sure? I see it when I build the doc locally. Although I don't understand why it doesn't show to be part of the Gudhi namespace.

Reply:
Uh, yes, I do see it now. Was it that it didn't appear yesterday? Or I was confused with Extended_filtration_data, which indeed doesn't appear in the doc, but that's on purpose, it is opaque and users just have to know to pass it from one function to the other.